Выполнено 3 задание

LoinRe opened the pull request.

LoinRe committed:
Реализована обработка команд в Telegram-боте.
-Дополнен класс TelegramBotUpdatesListener
-Подключен UserRepository для поиска пользователей в таблице USERS H2-базы
-Реализовано получение рекомендаций через RecommendationService и отправка их пользователю.

Labro0cd committed:
Task-3-Kirill-add-RuleStats

UlinSergey committed:
Добавил контроллер для получения статистики срабатываний динамических правил.

DanilIsaev committed:
Реализован ManagementController

DanilIsaev committed:
Реализован ManagementControllerTest

romatrix89 commented on src/main/java/projectwork/starbank/bot/TelegramBotUpdatesListener.java:
```diff
@@ -0,0 +1,81 @@
+
+    @Override
+    public int process(List<Update> updates) {
+        updates.forEach(update -> {
```
У этого метода очень сложная структура. Много вложенных if и такое ветвление логики очень тяжело проверять. Я бы порекомендовал разбить этот метод на несколько и упростить условные операторы. Т.е. в основном методе останется общая логика ветвления, но она будет вызывать другие методы, и за счет этого не будет ощущаться громоздкой и запутанной.

LoinRe committed:
Рефакторинг класса TelegramBotUpdatesListener:
- вынесена логика команд /start и /recommend в отдельные методы
- улучшена читаемость метода process()
